Pipfile

```diff
@@ -8,16 +8,15 @@ click = "*"
 sentry-sdk = "*"
 
 [dev-packages]
-bandit = "*"
 black = "*"
-coverage = "*"
```

Comment:
Don't we still need `coverage`? It's called in the `Makefile` test commands

Reply:
It appears to be installed alongside `coveralls`. I made this change when I noticed it wasn't required for the `carbon` repo. Would you prefer `coverage` to remain in `Pipfile`? 🤔

Reply:
Ah, disregard! 🙂